server/src/metadata/workspace-cache-version/workspace-cache-version.entity.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+
+@Entity('workspaceCacheVersion')
+export class WorkspaceCacheVersionEntity {
```

Comment:
Maybe we can add a unicity constrain on workspaceId?

Reply:
Actually I already put a unicity constraint on `workspaceId`

Reply:
Ah my bad, I forgot we could do that in the column decorator directly and not @Unique